bot/converters.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
What exactly relies on this?

Reply:
The SourceType return typehint in SourceConverter is imported conditionally under TYPE_CHECKING to prevent a cycle